quipucords/api/migrations/0005_auto_20190325_1756.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-03-25 17:56
```

Comment:
i would rename this migration like `0005_update_max_conccurency.py` or something more relevant to max-concurrency

Reply:
I decided to go with `0005_update_max-concurrency.py`

Reply:
:trollface: 😂